<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayersDockWidget</class>
 <widget class="QDockWidget" name="LayersDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>271</height>
   </rect>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>Layers</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QCheckBox" name="segmentedImage">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Display the Segmented Image</string>
      </property>
      <property name="text">
       <string>Segmented Image</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Segmented Opacity</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="opacitySpinner">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Adjust the opacity of the segmented image. 0 makes the segmented image opaque. 100 makes the segmented image transparent </string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QComboBox" name="compositeModeCB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Select the type of compositing of the original and segmented images</string>
      </property>
      <property name="statusTip">
       <string>Select the type of compositing of the original and segmented images</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QSlider" name="opacitySlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Adjust the opacity of the segmented image. 0 makes the segmented image opaque. 100 makes the segmented image transparent </string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Image Display</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="originalImage">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Display the original Image</string>
      </property>
      <property name="text">
       <string>Original Image</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QCheckBox" name="useColorTable">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Display Segmentation with Colors</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
